include/capypdf.hpp

```diff
@@ -483,6 +487,27 @@ class RasterImage : public CapyC<CapyPDF_RasterImage> {
     RasterImage(CapyPDF_RasterImage *ri) { _d.reset(ri); }
 };
 
+class ImageProperties : public CapyC<CapyPDF_ImagePdfProperties> {
```

Comment:
The C++ class name should be "the same" as the C one, in this case `ImagePdfProperties`.

A plain `ImageProperties` would mean "properties of the image itself" whereas this is "additional PDF document model properties to be used when the image is inserted".